### ISSUE TITLE
Set a port forwarding rule in the windows firewall automatically

### ISSUE THREAD

**Issue by MeijeSibbel** [Storj/storjshare-gui#563]
### Versions

Replace the values below with your own:

- **GUI Version:** v5.0.0
- **Operating System:** windows

### Expected Behavior

Please describe the program's expected behavior.

```
Currently most windows farmers that either use the basic connections (UPnP)/tunneling or even the advanced setup with TCP port forwarding all suffer from the same issue: Windows firewall since the last few updates blocks practically all incoming and outgoing traffic. One can add a exception for a specific port manually, this process should be automated so that Storj opens the specific port for a certain node automatically. 
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
Storj traffic is blocked by the windows firewall, manual adding a exception is necessary. 
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. ...
2. ...
3. ...

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot.

**Comment by MrFancyMonocle** [comment]
OS, version, and user-settings-specific things like this are a very low priority currently.
